src/main/java/com/tterrag/k9/mcp/SrgDatabase.java

```diff
@@ -65,6 +94,21 @@ public void reload() throws IOException {
                 String newOwner = mapping.getOwner();
                 if (existing == null || (owner != null && newOwner != null && owner.length() > newOwner.length())) {
                     srgs.put(mapping.getType(), mapping.getSRG(), mapping);
+                    if (mapping.getType() == MappingType.METHOD && mapping.getSRG().startsWith("func_")){
```

Comment:
Could this maybe be abstracted inside the factory?

Reply:
In what sense? It needs access to the static_methods list of the DB